m46x/StdDriver/drv_emac/synopGMAC_Dev.c

```diff
@@ -1645,8 +1645,8 @@ s32 synopGMAC_set_rx_qptr(synopGMACdevice * gmacdev, u32 Buffer1, u32 Length1, u
 #else
     DmaDesc * rxdesc = gmacdev->RxNextDesc;
 #endif
-    if(!synopGMAC_is_desc_empty(rxdesc))
-        return -1;
+	if(!synopGMAC_is_desc_empty(rxdesc))
+		return -1;
```

Comment:
is this right? I think you'd rather want to indent back the rest of the function instead

Reply:
Thanks of your advice, it's better to indent back the rest of the function.
Update it in c1ddf0ddc.